easybuild/easyconfigs/c/colorize/colorize-0.7.7-GCC-12.3.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'RubyGem'
+
+name = 'colorize'
+version = '0.7.7'
+
+homepage = 'https://github.com/fazibear/colorize'
+description = """Ruby gem for colorizing text using ANSI escape sequences.
+Extends String class or add a ColorizedString with methods to set the text color, background color and text effects."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+source_urls = ['https://rubygems.org/downloads/']
+sources = ['%(name)s-%(version)s.gem']
+checksums = ['d6ab95a5fcdea3c36c3327d38c1e79e2950ee1788506d8489ae35db330937a99']
+
+gem_file = '%(name)s-%(version)s.gem'
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
```

Comment:
`binutils` build dependency is not needed, it's already available when using `GCC` toolchain (not when using `GCCcore` as toolchain)

```suggestion
```